wrapper/src/main/java/software/amazon/jdbc/hostlistprovider/monitoring/ClusterTopologyMonitor.java

```diff
@@ -22,8 +22,9 @@
 import java.util.concurrent.TimeoutException;
 import org.checkerframework.checker.nullness.qual.Nullable;
 import software.amazon.jdbc.HostSpec;
+import software.amazon.jdbc.util.monitoring.Monitor;
 
-public interface ClusterTopologyMonitor extends AutoCloseable, Runnable {
+public interface ClusterTopologyMonitor extends Monitor, Runnable {
```

Comment:
Do you need to extend from Runnable?

Reply:
I think I can remove Runnable now, good catch